testsuite/example-deformer/osldeformer.cpp

```diff
@@ -284,7 +289,14 @@ main(int argc, char* argv[])
 
         // Run the shader (will automagically optimize and JIT the first
         // time it executes).
-        shadsys->execute(*ctx, *mygroup.get(), /*shade point index= */ i,
+
+        if(this_threads_index == uninitialized_thread_index) {
+                
+                this_threads_index = next_thread_index.fetch_add(1u);
+            }
+            int thread_index = this_threads_index;
+
+        shadsys->execute(*ctx, *mygroup.get(), thread_index, /*shade point index= */ i,
```

Comment:
Some strange indentation and spacing here. I guess this is part of testsuite so it's not run through clang-format? But it should still be cleaned up a bit.